dot/digest/digest.go

```diff
@@ -18,8 +18,8 @@ import (
 var maxUint64 = uint64(math.MaxUint64)
 
 var (
-	_      services.Service  = &Handler{}
-	logger log.LeveledLogger = log.NewFromGlobal(log.AddContext("pkg", "digest")) // TODO: add to config options (#1851)
+	_      services.Service = &Handler{}
+	logger                  = log.NewFromGlobal(log.AddContext("pkg", "digest")) // TODO: add to config options (#1851)
```

Comment:
You may want to keep it as an interface here to hide the actual implementation. I think `log.NewFromGlobal` returns a `*log.Logger` and not an interface. Not that it matters that much though

Reply:
I removed it cause I got the error:
```
dot/digest/digest.go:65:8: logger.Patch undefined (type log.LeveledLogger has no field or method Patch)
```

other packages don't seem to use the interface either